screens/HomeScreen.tsx

```diff
@@ -323,6 +364,39 @@ const HomeScreen = ({
               }
               keyExtractor={(item, index) => `${index}`}
             />
+            <Spacer />
+            {showBlacklist && (
+              <T center onPress={() => setIsRevealed(!isShowingBlacklist)}>
+                {isShowingBlacklist ? "Collapse" : "Reveal"}{" "}
```

Comment:
I think `Show` and `Hide` will be better verbs to use here.  Simpler words for non-native-English speakers.